ext/opcache/config.m4

```diff
@@ -70,30 +68,29 @@ if test "$PHP_OPCACHE" != "no"; then
     "])
 
     dnl Find out which ABI we are using.
-    case $host_alias in
-      x86_64-*-darwin*)
+    AS_CASE([$host_alias],
+      [x86_64-*-darwin*], [
         IR_TARGET=IR_TARGET_X64
         DASM_FLAGS="-D X64APPLE=1 -D X64=1"
         DASM_ARCH="x86"
-        ;;
-      *x86_64*|amd64-*-freebsd*)
+      ],
+      [*x86_64*|amd64-*-freebsd*], [
         IR_TARGET=IR_TARGET_X64
         DASM_FLAGS="-D X64=1"
         DASM_ARCH="x86"
-        ;;
-      i[[34567]]86*)
+      ],
+      [[i[34567]86*]], [
```

Comment:
quick question: can this condition and the next be merged in one ?

Reply:
Of course. Yes, good catch finding this one. Joined in the next commit.